maintainers/maintainer-list.nix

```diff
@@ -18400,6 +18400,11 @@
     githubId = 56157634;
     name = "Ruben Hönle";
   };
+  rubikcubed = {
+    github = "rubikcubed";
+    githubId = 91467402;
+    name = "rubikcubed";
+  };
```

Comment:
Do you think you could squash the changes to the maintainer list into a single commit, `maintainers: add rubikcubed`?

On a side note since you're new: be careful not to rebase commits past your own. It tends to cause a mass ping, and would require you to reopen your PR. If you're not confident in your git fu, you might want to mark the PR as draft first. It will mitigate the mass ping despite an erroneous rebase.